include/ccglib/common/arch.h

```diff
@@ -21,6 +30,11 @@ bool isCDNA3(cu::Device &device) {
           (arch.find("gfx942") != std::string::npos));
 }
 
+bool isRDNA4(cu::Device &device) {
+  const std::string arch(device.getArch());
+  return (arch.find("gfx11") != std::string::npos);
```

Comment:
```suggestion
  return (arch.find("gfx12") != std::string::npos);
```

Also not actually used in `hasFP8`, presumably because it is not yet implemented for RDNA4. Perhaps add a comment why `isRDNA4` is not used?